### PR TITLE
[Satisfactory] Temporary Fixes for 1.0 Release

### PR DESCRIPTION
**Pull request by gOOvER**
# Description
- updated for Satisfactory 1.0
- removed query and beacon port. Only 1 port is needed
- removed all setting from variables, which are now part of the new 1.0 Serverinterface ingame

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [x] You verify that the start command applied does not use a shell script
  * [ ] If some script is needed then it is part of a current yolk or a PR to add one
* [x] The egg was exported from the panel

### CODE REVIEW

File: satisfactory/egg-pterodactyl-satisfactory.json [hunk 1]
```diff
@@ -1,55 +1,35 @@
...
-        "update_url": null,
-        "version": "PTDL_v2"
+        "version": "PTDL_v2",
+        "update_url": null
     },
-    "exported_at": "2024-06-01T00:04:44+00:00",
+    "exported_at": "2024-09-10T17:58:10+02:00",
     "name": "Satisfactory",
-    "author": "red_thirten@yahoo.com",
+    "author": "rehlmgaming@gmail.com",
```
Comment (parkervcp) [inline satisfactory/egg-pterodactyl-satisfactory.json:9]: did you replace the author on purpose?

**Comment by lilkingjr1** [comment]
@gOOvER Would you mind too terribly if I PR'ed my own update? There's some things I'd like to fix and some polish I'd like to apply. Also, the README needs updating. However, if the email is fixed, everything else here looks like it will technically work, even if some of it is technically wrong. So, if we want to push something really quick like this, that's fine.

File: satisfactory/egg-pterodactyl-satisfactory.json (hunk 1 above)
Comment (gOOvER) [reply satisfactory/egg-pterodactyl-satisfactory.json:9]: its an old email :D

**Comment by gOOvER** [comment]
depreciated settings?

**Pull request merged**